Helper/Requests.php

```diff
@@ -337,25 +337,19 @@ public function buildBrowserData($request = [])
      */
     public function buildThreeDS2Data($additionalData, $storeId, $request = [])
     {
-        if ($this->adyenHelper->isCreditCardThreeDS2Enabled($storeId)) {
-            $request['additionalData']['allow3DS2'] = true;
-            $request['origin'] = $this->adyenHelper->getOrigin($storeId);
-            $request['channel'] = 'web';
-            $request['browserInfo']['screenWidth'] = $additionalData[AdyenCcDataAssignObserver::SCREEN_WIDTH];
-            $request['browserInfo']['screenHeight'] = $additionalData[AdyenCcDataAssignObserver::SCREEN_HEIGHT];
-            $request['browserInfo']['colorDepth'] = $additionalData[AdyenCcDataAssignObserver::SCREEN_COLOR_DEPTH];
-            $request['browserInfo']['timeZoneOffset'] = $additionalData[AdyenCcDataAssignObserver::TIMEZONE_OFFSET];
-            $request['browserInfo']['language'] = $additionalData[AdyenCcDataAssignObserver::LANGUAGE];
-
-            if ($javaEnabled = $additionalData[AdyenCcDataAssignObserver::JAVA_ENABLED]) {
-                $request['browserInfo']['javaEnabled'] = $javaEnabled;
-            } else {
-                $request['browserInfo']['javaEnabled'] = false;
-            }
+        $request['additionalData']['allow3DS2'] = true;
+        $request['origin'] = $this->adyenHelper->getOrigin($storeId);
+        $request['channel'] = 'web';
+        $request['browserInfo']['screenWidth'] = $additionalData[AdyenCcDataAssignObserver::SCREEN_WIDTH];
+        $request['browserInfo']['screenHeight'] = $additionalData[AdyenCcDataAssignObserver::SCREEN_HEIGHT];
+        $request['browserInfo']['colorDepth'] = $additionalData[AdyenCcDataAssignObserver::SCREEN_COLOR_DEPTH];
+        $request['browserInfo']['timeZoneOffset'] = $additionalData[AdyenCcDataAssignObserver::TIMEZONE_OFFSET];
+        $request['browserInfo']['language'] = $additionalData[AdyenCcDataAssignObserver::LANGUAGE];
+
+        if ($javaEnabled = $additionalData[AdyenCcDataAssignObserver::JAVA_ENABLED]) {
+            $request['browserInfo']['javaEnabled'] = $javaEnabled;
         } else {
-            $request['additionalData']['allow3DS2'] = false;
-            $request['origin'] = $this->adyenHelper->getOrigin($storeId);
-            $request['channel'] = 'web';
+            $request['browserInfo']['javaEnabled'] = false;
         }
```

Comment:
Not 100% connected to this ticket but do we actually still need to add it or is the stateData enough since it's coming from the frontend in the state data?

Reply:
I noticed that this method is not called anywhere in the code, and changing some of the variables here did not affect the data in the outgoing API call. but I'm not sure whether there's something I'm missing.

Reply:
@acampos1916 what do you think? do you know if we can safely remove the `buildThreeDS2Data()` method?

Reply:
It was being used before moving to v7, so it can be removed now @peterojo